### DISCUSSION TITLE
How to name license of Mastodon logo

### DISCUSSION THREAD

**Discussion by buhtz** [mastodon/mastodon#33352] [Q&A]
Hello,
I would like to use the Mastodon logo to point to my own account in my README.md. But I don't want to use a remote URL but copy the file in my repo and use it from there. But I need license information in this case.

https://joinmastodon.org/de/branding

I don't see license info there. What are this logos?

**Comment by pReya** [comment]
Same question here.

**Reply by juh2** [reply]
As far as I can see you submit yourself to the US legislation, which you should keep in mind concerning the current administration. https://joinmastodon.org/de/trademark

"You further agree that any dispute arising in connection with your use of our trademarks or under these terms and conditions shall be under the exclusive jurisdiction of the state and federal courts of New York in the United States of America and that the state and federal courts of New York shall have personal jurisdiction over you for the purposes of adjudicating any dispute concerning the use of our trademarks or these terms and conditions."
